src/app/shared/components/footer/footer.component.scss

```diff
@@ -9,51 +9,72 @@ footer {
 
         .footer-list {
             display: flex;
-            flex-direction: row;
-            justify-content: flex-start;
+            flex-wrap: wrap;
+            justify-content: space-between;
+            align-items: stretch;
             padding: 8px;
+            filter: brightness(0) invert(1);

```

Comment:
You don't want to use a filter to color text, this will make it render in a weird way and it's not ideal.